liquigraph-examples/dagger2/src/test/java/org/liquigraph/examples/dagger2/ApplicationTest.java

```diff
@@ -15,24 +15,28 @@
  */
 package org.liquigraph.examples.dagger2;
 
+import java.io.IOException;
+import java.net.ServerSocket;
 import org.junit.ClassRule;
 import org.junit.Test;
 import org.neo4j.harness.junit.Neo4jRule;
-
-import java.io.IOException;
-import java.net.ServerSocket;
+import org.slf4j.bridge.SLF4JBridgeHandler;
 
 import static io.restassured.RestAssured.get;
 import static java.util.Arrays.stream;
 import static org.hamcrest.Matchers.containsString;
 
 public class ApplicationTest {
 
+    static {
+        SLF4JBridgeHandler.removeHandlersForRootLogger();
+    }
+
     @ClassRule
-    public static Neo4jRule neo4j = withVersionAwareConfig(new Neo4jRule());
+    public static final Neo4jRule neo4j = withVersionAwareConfig(new Neo4jRule());
 
     @Test
-    public void service_responds_after_migration() throws Exception {
+    public void service_responds_after_migration() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)